src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/project.json

```diff
@@ -1,5 +1,11 @@
 {
-  "description": "ASP.NET Identity providers that use Entity Framework.",
+  "description": "ASP.NET Core Identity providers that use Entity Framework Core.",
```

Comment:
I wonder why we were using plural "providers".  I think of this package as containing just one "provider" which contains a set of store classes.